### DISCUSSION TITLE
（广告）光圈科技来六分街招人啦！

### DISCUSSION THREAD

**Discussion by nobuts-wt** [share121/inter-knot#2238] [其他]
女士们，先生们，欢迎你们来到光圈。无论是宇航员、战争英雄、奥运选手——你们即将来到这里，因为我们想要最好的，而你们就是我们。那么：谁准备好做一些科学研究呢？先介绍一下我，我是**凯夫·约翰逊**。我拥有整个**光圈科技**，在我们的浓缩领域，每天都有一千项测试。我不能亲自监督他们中的每一个人，所以这些预先录制的信息将涵盖你可能有的任何问题，并回应你的科学冒险过程中可能发生的任何事件。你的测试任务也会有所不同，这取决于你让世界屈服于你的意愿的方式。
顺便说一句，所有这些科学领域都是由石棉制成的。将老鼠拒之门外。如果您感到呼吸急促、持续干咳或心脏停止跳动，请告诉我们。因为那不是测试的一部分。那是**石棉**。好消息是，实验室的男孩们说，**石棉中毒的症状显示中位潜伏期为四十四点六年**，所以如果你三十岁或更老，你就是在笑。最坏的情况是，你错过了几轮 _canasta_，再加上你把科学事业推进了三个世纪。我把这些数字输入我的计算器，它让人感到快乐。
科学不是关于为什么。。为什么我们的科学有这么多危险？如果你如此热爱它，为什么不嫁给安全的科学呢。_事实上，为什么不发明一种特殊的安全门？_它不会在出去的时候撞到你的屁股，因为你被解雇了。__
_不是你，测试对象，你做得很好，_
_是的，你，拎好行李箱，出前门，去停车场打辆汽车。再见_
       

（_好吧，我一直在想。当生活给你柠檬时？不要做柠檬水。让生活夺回柠檬！生气吧！“我不想要你那该死的柠檬！我该怎么处理这些呢？_）
(_我就是那个要烧你房子的人！
![Cave_Johnson_Painting_3](https://github.com/user-attachments/assets/470a6ca1-4c22-4c46-a410-4366bda9c330)

用柠檬！我要让我的工程师发明一种可燃的柠檬，可以烧毁你的房子！_)

作者声明：本文全部使用凯文 约翰逊的台词改编

**Comment by CAONIMA-5418** [comment]
申请加入光圈科技<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
